source/guides/stacks/mers.rst

```diff
@@ -21,19 +22,34 @@ complex standard-compliant encoders, and optimizing across the
 hardware-software stack for efficiency are all engineering and time
 investments for developers.
 
-The Media Reference Stack (MeRS) offers a highly optimized software stack for
-Intel Architecture to enable media prioritized workloads, such as transcoding
-and analytics. |MERS| abstracts away the complexity of integrating multiple
-software components and specifically tunes them for Intel platforms. |MERS|
-allows media and visual cloud developers to deliver experiences using a simple
-containerized solution. 
+|MERS| abstracts away the complexity of integrating multiple software
+components and specifically tunes them for Intel platforms. |MERS| allows
+media and visual cloud developers to deliver experiences using a simple
```

Comment:
"deliver experiences" ? this is really vague. Is it an application? A media experience?